openpgp/packet/signature.go

```diff
@@ -416,6 +418,13 @@ func parseSignatureSubpacket(sig *Signature, subpacket []byte, isHashed bool) (r
 		if sigType := sig.EmbeddedSignature.SigType; sigType != SigTypePrimaryKeyBinding {
 			return nil, errors.StructuralError("cross-signature has unexpected type " + strconv.Itoa(int(sigType)))
 		}
+	case policyUriSubpacket:
+		// Policy URI, section 5.2.3.20
+		if !isHashed {
+			return
+		}
+		sig.PolicyURI = make([]byte, len(subpacket))
+		copy(subpacket, sig.PolicyURI)
```

Comment:
This `copy` is the wrong way around, so it doesn't actually currently work :sweat_smile:
(I think with a string, this would be slightly more fool-proof, see `RevocationReasonText` for inspiration.)

Reply:
done

Reply:
Thanks! Looks good, just one last nitpick; this copy is now no longer necessary.